docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -456,41 +452,61 @@ Consul API and UI can be accessed using the consul token (Secret ID). For the sn
 #### Changing TLS certificates
 The API Gateway setup generates a self-signed certificate with a short expiration by default.
 
-The default certificate is stored at `/var/snap/edgexfoundry/current/nginx/nginx.crt`.
-
-The default private key is stored at `/var/snap/edgexfoundry/current/nginx/nginx.key`.
-
 The JWT authentication token that is consumed by the proxy is sensitive and it is important that
 measures are taken to ensure that clients do not disclose the JWT to unauthorized parties.
 For this reason, the default certificate and key should be replaced
 with a certificate and key that is trusted by connecting clients.
-(This can be done a `sudo cp` command or equivalent.)
 
+The certificate and key can be replaced locally. They are located at:
+
+- `/var/snap/edgexfoundry/current/nginx/nginx.crt`
+- `/var/snap/edgexfoundry/current/nginx/nginx.key`
+
+Changes to the files should be followed by reloading Nginx: `sudo snap restart --reload edgexfoundry.nginx`
+
+Alternatively, the certificate and key can be replaced using the snapped `secrets-config` application. To get the usage help:
+```bash
+edgexfoundry.secrets-config proxy tls -h
+```
+Refer to the [secrets-config proxy](../../security/secrets-config-proxy/) documentation.
 
 !!! example
     Given the following files created outside the scope of this document:
     
-    * `nginx.crt` user-provided certificate (replacing the default)
-    * `nginx.key` user-provided private key (replacing the default)
+    * `server.crt` user-provided certificate (replacing the default)
+    * `server.key` user-provided private key (replacing the default)
     * `ca.pem` certificate authority file (that signed `nginx.crt`, directly or indirectly)
+  
+    available at a directory accessible to the snapped application (e.g. `/var/snap/edgexfoundry/common`), the certificate and key can be replaced as follows:
+
+    1. Add new certificate files:
+    ```bash
+    sudo edgexfoundry.secrets-config proxy tls \
+      --targetFolder /var/snap/edgexfoundry/current/nginx \
+      --inCert /var/snap/edgexfoundry/common/server.crt \
+      --inKey  /var/snap/edgexfoundry/common/server.key 
+    ```
+    2. Remove the temporary files
```

Comment:
I suggest add some details for this step:

```suggestion
    2. Remove the temporary files:
    ```bash
    sudo rm /var/snap/edgexfoundry/common/server.crt
    sudo rm /var/snap/edgexfoundry/common/server.key
    ```
```